resolver-dns/src/test/java/io/netty/resolver/dns/RoundRobinDnsAddressResolverGroupTest.java

```diff
@@ -0,0 +1,47 @@
+package io.netty.resolver.dns;
+
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.channel.socket.nio.NioDatagramChannel;
+import io.netty.resolver.InetSocketAddressResolver;
+import io.netty.resolver.RoundRobinInetAddressResolver;
+import org.junit.Test;
+
+import java.net.InetAddress;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class RoundRobinDnsAddressResolverGroupTest {
+
+    @Test
+    public void testClose() throws InterruptedException {
+        NioEventLoopGroup eventLoopGroup = new NioEventLoopGroup(1);
+
+        try {
+            RoundRobinDnsAddressResolverGroup roundRobinGroup =
+                    new RoundRobinDnsAddressResolverGroup(NioDatagramChannel.class,
+                            DefaultDnsServerAddressStreamProvider.INSTANCE);
+
+            InetSocketAddressResolver inetSocketAddressResolver =
+                    (InetSocketAddressResolver) roundRobinGroup.getResolver(eventLoopGroup.next());
+
+            RoundRobinInetAddressResolver roundRobinInetAddressResolver =
+                    (RoundRobinInetAddressResolver) inetSocketAddressResolver.nameResolver();
+
+            InflightNameResolver<InetAddress> inflightNameResolver =
+                    (InflightNameResolver<InetAddress>) roundRobinInetAddressResolver.nameResolver();
+
+            DnsNameResolver dnsNameResolver = (DnsNameResolver) inflightNameResolver.delegate;
+
+            assertTrue(dnsNameResolver.ch.isOpen());
+
+            roundRobinGroup.close();
+            dnsNameResolver.ch.closeFuture().await(2, TimeUnit.SECONDS);
```

Comment:
Should we just use `sync()` and use `@Test(timeout = 2000)` ?

Reply:
Sure!

Reply:
Done and pushed.